### DISCUSSION TITLE
js file to jsx format not work

### DISCUSSION THREAD

**Discussion by uptownboy7** [brackets-cont/brackets#116] [Q&A]
![ee3](https://user-images.githubusercontent.com/57980360/147842529-803b959e-2669-4b36-8960-8e4026396936.png)

if i open js file, its default is js Highlight even i clicked [set Default to js files] before.

![ww2](https://user-images.githubusercontent.com/57980360/147842484-8b13e821-5bc7-4b0e-96b0-3e1108ccb7f4.png)
 
i edit react [.js] files but it doesn't recognize as JSX when i selected [set Default to js files]
the first time i clicked the option, it works.
but when i close brackets, and reopen the brackets, it initialized!
JS highlighter shows but not JSX highlighter

It Doesn't Work.

plz help me.

**Comment by marcosbacon** [comment]
Yup. It actually should automatically identify the project as a React App, and then interpret any .js as .jsx. Anyone?
